source/geod24/LocalRest.d

```diff
@@ -180,7 +180,7 @@ private final class LocalScheduler : C.FiberScheduler
     private Waiting[ulong] waiting;
 
     /// Should never be called from outside
-    public override Condition newCondition(Mutex m = null) nothrow
+    public override Condition newCondition() nothrow
```

Comment:
A note on this: I originally removed that parameter because, within a `FiberScheduler`, there is no  need for a `Mutex`, all fibers run in the same Thread. Mutexes are still necessary for cross-threads synchronization, so will be required by *some* channels.